lib/events.js

```diff
@@ -9,7 +9,8 @@ var events_table = $('#hack-night-events').dataTable( {
         null
     ],
     "bInfo": false,
-    "bPaginate": false
+    "bPaginate": true,
```

Comment:
I like the pagination - 450+ entries is getting way too long for one page.

However the style of the pagination controls is a little funky. We should use the CHN blue/red colors:

![Screen Shot 2021-09-02 at 8 49 23 AM](https://user-images.githubusercontent.com/919583/131855665-42bc60ec-d7e8-4978-845c-1e00f09ec680.png)

Reply:
I switched these to CHN colors